cwatch.php

```diff
@@ -1358,7 +1377,7 @@ private function getServiceRules(array $vars = null, $edit = false)
                             "/^([a-z0-9]|[a-z0-9][a-z0-9\-]{0,61}[a-z0-9])(\.([a-z0-9]|[a-z0-9][a-z0-9\-]{0,61}[a-z0-9]))+$/"
```

Comment:
Shouldn't the domain name be case-insensitive? This regex could include a "/i" for that. I suppose you may want to also update other locations to save the domain in lower-case.